InteractiveHtmlBom/ecad/schema/genericjsonpcbdata_v1.schema

```diff
@@ -0,0 +1,574 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$ref": "#/definitions/GenericJSONPCBData",
+    "definitions": {
+        "GenericJSONPCBData": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "_type": {
+                    "type": "string"
```

Comment:
This doesn't do anything ?

Reply:
_type was to make it extremely unambiguous with an EasyEDA file, but I've removed this from the JSON and will just look for the existence of a "pcbdata" object instead